src/sentry/conf/server.py

```diff
@@ -3510,9 +3511,6 @@ def build_cdc_postgres_init_db_volume(settings: Any) -> dict[str, dict[str, str]
     KAFKA_SHARED_RESOURCES_USAGE: {"cluster": "default"},
 }
 
-
-# If True, consumers will create the topics if they don't exist
-KAFKA_CONSUMER_AUTO_CREATE_TOPICS = True
```

Comment:
Why is this removed ?

Reply:
I cleaned it up since it doesn't do anything. Moved into https://github.com/getsentry/sentry/pull/65891